etc/nginx/prod/nginx.conf

```diff
@@ -86,7 +86,7 @@ http {
 
   server {
     listen 1080;
-    server_name alpha.policycompass.eu www.policycompass.eu policycompass.eu;
+    server_name policycompass.eu www.policycompass.eu;
```

Comment:
I tried it that way, but then the two domains are not separate. Everything is redirected then to the first server name listed.